.github/workflows/python.yaml

```diff
@@ -127,6 +127,12 @@ jobs:
         . .github/workflows/activate_miniconda.sh
         scripts/conda_build.sh --output-folder $HOME/build/
 
+    - name: Build conda Packages for Katana Metagraph Plugin
+      run: |
+        . .github/workflows/activate_miniconda.sh
+        cd metagraph-plugin/conda_recipe
+        conda-build . -c conda-forge -c metagraph -c katanagraph/label/dev --output-folder $HOME/build/
```

Comment:
Instead of `-c katanagraph/label/dev` which will get the katana packages that are currently on anaconda, this should use `-c file://$HOME/build/`. That will use the packages built in the previous step by using those newly built packages as a channel.

Reply:
Fixing this is *critical*. If we do not, this could fail to allow changes that affect both `katana` and `metagraph_katana` from passing CI since for this test to pass, `matagraph_katana` needs to match the anaconda.org katana package. But to actually be correct w.r.t. the changes to `katana`, `matagraph_katana` will need to be changed in ways that are not compatible with those anaconda.org packages.